json_object.c

```diff
@@ -1265,3 +1265,128 @@ int json_object_equal(struct json_object* jso1, struct json_object* jso2)
 	return 0;
 }
 
+static int json_object_copy_serializer_data(struct json_object *src, struct json_object *dst)
+{
+	/* `src` and `dst` were validated by now ==> no null check for them */
+	if (src->o_type != json_type_double)
```

Comment:
so, regarding other/external serializers, maybe a `_user_copy` function would be needed